__tests__/pet.test.js

```diff
@@ -0,0 +1,102 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+    it('returns an object', () => {
+        expect(new Pet('Dexter')).toBeInstanceOf(Object);
+    });
+});
+describe('naming pet', () => {   
+    it('sets the name property', () => {
+        const pet = new Pet('Dexter');
+        expect(pet.name).toEqual('Dexter');
+    });
+});
+describe('initial age', () => {
+    it('has a initial age of 0', () => {
+        const pet = new Pet('Dexter');
+        expect(pet.age).toEqual(0);
+    });
+});
+describe('growUp', () => {
+    it('increases the age by 1, increases hunger by 5 and dcreases fitness by 3', () => {
+        const pet = new Pet('Dexter');
+        pet.growUp();
+        expect(pet.age).toEqual(1);
+        expect(pet.hunger).toEqual(5);
+        expect(pet.fitness).toEqual(7);
+    });
+    it('throws an error if the pet is not alive', () => {
+        const pet = new Pet ('Dexter');
+        pet.age = 30;
+        expect(() => pet.growUp()).toThrow('Your pet is no longer alive :(');
+    });
+});
+describe('initial hunger', () => {
+    it('has a initial hunger of 0', () => {
+        const pet = new Pet('Dexter');
+        expect(pet.hunger).toEqual(0);
+    });
+});
+describe('initial fitness', () => {
+    it('has a initial fitness of 10', () => {
+        const pet = new Pet('Dexter');
+        expect(pet.fitness).toEqual(10);
+    });
+});
+describe('play',() => {
+    it('plays with the pet and increases the fitness by 4 to a maximum of 10', () => {
+        const pet = new Pet('Dexter');
+        pet.fitness = 8;
+        pet.play();
+        expect(pet.fitness).toEqual(10);
```

Comment:
Its a common convention to leave a clear line before your assertions in test blocks to help a bit with readability. Not a hard and fast rule but it makes sense if you have a large test or multiple assertions